cloud/src/rate-limiter/rate_limiter.h

```diff
@@ -35,12 +38,32 @@ class RateLimiter {
 public:
     RateLimiter() = default;
     ~RateLimiter() = default;
+
     void init(google::protobuf::Service* service);
+
     std::shared_ptr<RpcRateLimiter> get_rpc_rate_limiter(const std::string& rpc_name);
 
+    void for_each_rpc_limiter(
```

Comment:
add comment even though it seems simple, describe: behavior, assumptions, params and return value